Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -716,6 +716,16 @@
     ],
     "initial": "none"
   },
+  "clip-rule": {
+    "affects-layout": false,
+    "animation-type": "discrete",
+    "inherited": true,
+    "initial": "nonzero",
+    "valid-identifiers": [
+      "nonzero",
+      "evenodd"
+    ]
```

Comment:
I think both this and `fill-rule` should be using `"valid-types": [ "fill-rule" ]` instead, because we already have an enum defined for that.

Reply:
Sure, that works :+1: Put the `fill-rule` change in its own commit (since it's somewhat unrelated).